lab5/Parser.py

```diff
@@ -62,7 +62,10 @@ def success(self):
         self.conf.s = States.F
 
     def expand(self):
-        pass
```

Comment:
I liked this pass, it was my favorite one ;(

Reply:
Sorry